### PR TITLE
refactor: Final tweaks to App Services examples for V2

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/Contributing.md.

## What is the current behavior?
makefiles need clean-up 
READMEs need clean-up
Don't need more than one Docker example

## What is the new behavior?
Only one example needs to have Docker example
Clean-up READMEs and makefiles

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: application-services/README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 ## Overview
 
-This folder contains various examples of Application Services based on the App Functions SDK and Profiles that can be used with App Service Configurable. See the [Application Services](https://docs.edgexfoundry.org/1.2/microservices/application/ApplServices/.) documentation for complete details on Application services.
+This folder contains various examples of Application Services based on the `Ireland 2.0.x` App Functions SDK and Profiles that can be used with App Service Configurable. See the [Application Services](https://docs.edgexfoundry.org/1.2/microservices/application/ApplServices/.) documentation for complete details on Application services.
```
Comment (jim-wang-intel) [inline application-services/README.md:5]: Should we use `https://docs.edgexfoundry.org/2.0/` as the version for Docs?

File: application-services/custom/custom-trigger/README.md [hunk 2]
```diff
@@ -8,4 +8,8 @@ In this example we introduce a trigger that listens for input on os.Stdin and se
 
 To run:
 
-`make docker && docker run -it edgexfoundry/docker-custom-trigger-example:dev`
\ No newline at end of file
+```
```
Comment (jim-wang-intel) [inline application-services/custom/custom-trigger/README.md:11]: console?
```suggestion
```console
```

File: application-services/custom/secrets/README.md [hunk 3]
```diff
@@ -2,119 +2,85 @@
...
 
-3. Use docker exec to run commands on the running vault container instance.
+   ```
+   docker-compose -p edgex ps
+   ```
 
-   `docker exec -it <container_id> /bin/sh`
+   Which will output the following:
+
+   ```bash
```
Comment (jim-wang-intel) [inline application-services/custom/secrets/README.md:55]: use console probably looks better
```suggestion
   ```console
```

File: application-services/custom/secrets/README.md [hunk 4]
```diff
@@ -131,5 +97,12 @@ These tests use a collection of Postman requests, in *SecretsExample.postman_col
 
 4. View the service's logs to verify that the secrets were retrieved. We'll view the secrets in the application's console (in production, NEVER log your application's secrets. This is done in the example service to demonstrate the functionality).
 
-   1. Running in docker: `docker logs app-service-secrets-example`
-   2. Running natively: view the console
+   ```bash
```
Comment (jim-wang-intel) [inline application-services/custom/secrets/README.md:100]:
```suggestion
   ```console
```

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]
some minor tweaks

File: application-services/README.md (hunk 1 above)
Comment (lenny-intel) [reply application-services/README.md:5]: Good catch. Thought I had got all those. ;-)

File: application-services/custom/custom-trigger/README.md (hunk 2 above)
Comment (lenny-intel) [reply application-services/custom/custom-trigger/README.md:11]: Humm, not sure... ;-)

File: application-services/custom/secrets/README.md (hunk 3 above)
Comment (lenny-intel) [reply application-services/custom/secrets/README.md:55]: Oh, didn't know that was an option... will try it.

**Review by AlexCuse — DISMISSED** [review]

File: application-services/custom/secrets/README.md (hunk 3 above)
Comment (lenny-intel) [reply application-services/custom/secrets/README.md:55]: console isn't an option.

File: application-services/custom/custom-trigger/README.md (hunk 2 above)
Comment (lenny-intel) [reply application-services/custom/custom-trigger/README.md:11]: console not a valid option.

File: application-services/custom/secrets/README.md (hunk 4 above)
Comment (lenny-intel) [reply application-services/custom/secrets/README.md:100]: consule not a valid option

Comment (jim-wang-intel) [reply application-services/custom/secrets/README.md:100]:
```console
$ test console
```

Comment (jim-wang-intel) [reply application-services/custom/secrets/README.md:100]: the original code:
`
#   ```console
   $ test console
   ```
`

Comment (jim-wang-intel) [reply application-services/custom/secrets/README.md:100]: so to me it is an option at here :-)

Comment (lenny-intel) [reply application-services/custom/secrets/README.md:100]: Using console only make text blue , wahhoo!
``` console
make build
./app-service
```

**Review by jim-wang-intel — DISMISSED** [review]

File: application-services/custom/secrets/README.md (hunk 3 above)
Comment (lenny-intel) [reply application-services/custom/secrets/README.md:55]: adda

File: application-services/custom/custom-trigger/README.md (hunk 2 above)
Comment (lenny-intel) [reply application-services/custom/custom-trigger/README.md:11]: added

File: application-services/custom/secrets/README.md (hunk 4 above)
Comment (lenny-intel) [reply application-services/custom/secrets/README.md:100]: added

**Review by jim-wang-intel — APPROVED** [review]
LGTM
Thanks for efforts on trying that out! 👍

**Pull request merged**